docs-ref-conceptual/azure-cli-vm-tutorial-3.md

```diff
@@ -38,14 +41,33 @@ az vm create \
   --verbose 
 ```
 
+# [PowerShell](#tab/powershell)
+
+```azurecli-interactive
+# Create PowerShell variable
+$vmName = "TutorialVM1"
+
+az vm create `
+    --resource-group $resourceGroup `
+    --name $vmName `
+    --image Ubuntu2204 `
+    --vnet-name $vnetName `
+    --subnet $subnetName `
+    --generate-ssh-keys `
+    --output json `
+    --verbose
+```
+
+---
+
 > [!NOTE]
 > If you have an SSH key named `id_rsa` already available, this key is used for authentication rather than having a new
 > key generated.
 
 As the VM is created, you see the local values used and Azure resources being created due to the `--verbose` option.
 Once the VM is ready, a JSON is returned from the Azure service including the public IP address.
 
-```json
+```output
```

Comment:
@v-abdriaz , output code blocks should always have an `output` code tag regardless of table output, json, etc.  See Contributor's Guide [Output code blocks](https://review.learn.microsoft.com/en-us/help/contribute/conventions-azure-cli?branch=main#output).